docs/react native/androidIntegration.md

```diff
@@ -8,19 +8,37 @@ permalink: docs/react-native/android-integration/
 ---
 
 # Android Integration
-
 {: .no_toc }
 
 To integrate the Android SDK's within your React Native app, please use the following steps.
```

Comment:
We could reword it if removing the section below "To integrate the Android SDK's within your React Native app, open the Android folder inside your React Native project using Android Studio and follow the steps listed below."